src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingConfigValidator.java

```diff
@@ -0,0 +1,14 @@
+package com.snowflake.kafka.connector.internal.streaming;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+/** Validates connector config for Snowpipe Streaming */
+public interface StreamingConfigValidator {
```

Comment:
Do you plan to create more validators? As for now It looks like these new interfaces are not really usefull.

Reply:
I would say they might be useful for testing purpose.